lib/mongo/server/monitor/connection.rb

```diff
@@ -131,7 +133,7 @@ def initialize(address, options = {})
         #
         # @since 2.0.0
         def timeout
-          @timeout ||= options[:connect_timeout] || CONNECT_TIMEOUT
+          @timeout ||= options[:connect_timeout]
```

Comment:
I put the constant in Server so that Address and Server::Monitor::Connection could reference it.